apypie/api.py

```diff
@@ -53,7 +53,9 @@ def __init__(self, **kwargs):
         self._session.verify = kwargs.get('verify_ssl', True)
 
         self._session.headers = {
-            'Accept': 'application/json;version={}'.format(self.api_version)
+            'Accept': 'application/json;version={}'.format(self.api_version),
+            'Accept-Encoding': 'deflate, gzip',
```

Comment:
The docs say this should be default. Looks like we're clearing the default headers this way:
```
In [8]: requests.Session().headers                                                                                                                                                    
Out[8]: {'User-Agent': 'python-requests/2.21.0', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive'}
```

We should probably use `self._session.headers['User-Agent'] = 'apypie (https://github.com/Apipie/apypie)'` instead

Reply:
updated